src/Symfony/Component/Security/Core/Tests/Authorization/AccessDecisionManagerTest.php

```diff
@@ -39,36 +75,63 @@ public function testVoterCalls()
             $this->getUnexpectedVoter(),
         ];
 
-        $strategy = new class implements AccessDecisionStrategyInterface {
-            public function decide(\Traversable $results): bool
-            {
-                $i = 0;
-                foreach ($results as $result) {
-                    switch ($i++) {
-                        case 0:
-                            Assert::assertSame(VoterInterface::ACCESS_DENIED, $result);
-
-                            break;
-                        case 1:
-                            Assert::assertSame(VoterInterface::ACCESS_GRANTED, $result);
-
-                            return true;
+        if($useVoteObject) {
```

Comment:
same question: can't we do without this flag?
```suggestion
        if ($useVoteObject) {
```